tests/common.go

```diff
@@ -8853,3 +8980,9 @@ func DeleteCrAndRepo(appData *asyncdr.AppData, appPath string) error {
 	asyncdr.DeleteCRAndUninstallCRD(appData.OperatorName, appPath, appData.Ns)
 	return nil
 }
+
+// Set default provider as aws
+func getClusterProvider() string {
+	clusterProvider = os.Getenv("CLUSTER_PROVIDER")
+	return clusterProvider
+}
```

Comment:
Not required as we are passing this as environment variable

Reply:
Added this to make it more readable